src/stores/activityStore/index.js

```diff
@@ -1,6 +1,6 @@
 import { observable, action } from 'mobx'
 
-class EventStore {
+class ActivityStore {
```

Comment:
Is it React practice to use `camelCase` for file/folder names (`activityStore` in this case)?

Note this is only a question.

Reply:
The answer is YES.

Reply:
Thanks. Then another question is, why subfolders under `components` and `containers` are capitalized `CamelCase`?

Reply:
In react project, file/dir name should be named in `CamelCase`, except the `Component` should be capitalized.
https://stackoverflow.com/questions/30373343/reactjs-component-names-must-begin-with-capital-letters